versioned_docs/version-8.5/components/connectors/protocol/soap.md

```diff
@@ -0,0 +1,211 @@
+
+### WSS signature
+
+Use the **WSS signature** in the **Authentication** dropdown when the requested SOAP endpoint requires a message to be
+cryptographically signed with a [signature](http://docs.oasis-open.org/wss-m/wss/v1.1.1/cs01/wss-SOAPMessageSecurity-v1.1.1-cs01.html#_Toc307407954).
+
+Enter all necessary fields according to your service specification.
+
+## SOAP Message
+
+### SOAP version
+
+Select the latest supported version of the SOAP service.
+
+### SOAPAction HTTP header
+
+Enter the SOAPAction HTTP header that will be used in the request. Leave this value blank if the SOAPAction HTTP header
+won't be used in our request.
+
+### SOAP header
+
+From the dropdown select whether the **SOAP header** is required, and if so, in which format you wish to provide it.
+
+### SOAP body
+
+From the **SOAP body** dropdown, select whether you will provide the SOAP request body in a form of **Template**, or
+**XML compatible JSON**.
+
+#### Template
+
+When **Template** is chosen, you have to enter the **XML template** value, e.g.
+`<camunda:Param><camunda:ParamType>{{paramValue}}</camunda:ParamType></camunda:Param>`.
+
+You have to enter the **XML template context** value, e.g.
+`={paramValue: 1234567890}`.
+
+You have to enter the **Namespaces** value, e.g.
+`={"camunda":"http://my.service.com/webservicesserver/"}`.
+
+#### XML compatible JSON
+
+When **XML compatible JSON** is chosen, you have to enter the **JSON definition**, e.g.
+
+```xml
+<camunda:Object01>
+  <camunda:Object02>{{myObjectValue}}</camunda:Object02>
+</camunda:Object01>
+```
+
+You have to enter the **Namespaces** value, e.g.
+`={"camunda":"http://my.service.com/webservicesserver/"}`.
+
+## Output mapping
+
+### Result variable
+
+You can export a complete response from an SOAP call into a dedicated variable accessible anywhere in a process.
+To do so, just input a variable name in the **Result variable** field. We recommend using a unique name to avoid
+variables being overwritten.
+
+## Usage examples
+
+### Example 1
+
+For example, you would like to send a following SOAP request:
+
+URL: `https://myservice:8888/webservice.wso`
+
+Body:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
+  <soap:Body>
+    <Object01>
+      <Object02>12345</Object02>
+    </Object01>
+  </soap:Body>
+</soap:Envelope>
+```
+
+In order to do so, in your BPMN diagram, set the field **Service URL** as `https://myservice:8888/webservice.wso`, and **SOAP body** as
+
+```json
+{
+  "Object01": {
+    "Object02": 12345
+  }
+}
+```
+
+### Example 2: pre-defined namespaces
+
+Consider a namespace is defined within your objects, and you wish to send the following request:
+
+URL: `https://myservice:8888/webservice.wso`
+
+Body:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
+  <soap:Body>
+    <Object01 mlns="http://www.my.namespace.com/namespace/">
```

Comment:
Same for this, is it mlns or xmlns? Im not a soap expert but flagging just incase